src/promptflow-tools/promptflow/tools/common.py

```diff
@@ -98,6 +98,16 @@ def parse_function_role_prompt(function_str):
                     " 'How to use functions with chat models' in our gallery.")
 
 
+def parse_common_role_prompt(role_prompt):
```

Comment:
rename as `parse_name_and_content` and merge the two logics (function role content and other role content) in the same function, we could have a pattern rule as param.